src/routes.json

```diff
@@ -98,6 +98,13 @@
         "title": "DMCA"
     },
     {
+        "name": "down-deep",
```

Comment:
do you mean the path to the page be scratch.mit.edu/camp or just change the `name` property to camp?